board.py

```diff
@@ -0,0 +1,50 @@
+class Board():
+
+    def __init__(self, board_size, number_of_dices, final_position, root_pos, number_of_players, players, leaderboard):
+        self.__board_size = board_size
+        self.__number_of_dices = number_of_dices
+        self.__final_position = final_position
+        self.__root_pos = root_pos
+        self.__leaderboard = leaderboard
+        self.__players = players
+        self.__number_of_players = number_of_players
+
+    # Get the whole Leader Board
+    def get_leaderboard(self,):
+        return self.__leaderboard
+
+    # Get the player whose turn is next
+    def get_player(self,i):
+        return self.__players[i]
+
+    # Get number of players left
+    def get_number_of_players(self):
+        return self.__number_of_players
+
+    # Get the position a player must go to if he lands on ith block
+    def get_final_pos(self,i):
+        return self.__final_position[i]
+
+    # Get the end position
```

Comment:
redundant comments at multiple places. The method name says the exact same thing